arkouda/plotting.py

```diff
@@ -50,3 +57,99 @@ def plot_dist(b, h, log=True, xlabel=None, newfig=True):
     plt.gca().set_title("cumulative distribution")
     if xlabel is not None:
         plt.gca().set_xlabel(xlabel, fontsize=14)
+
+
+def hist_all(ak_df: DataFrame, cols: list = [], log: bool = True):
+    """
+    Create a grid plot histogramming all numeric columns in ak dataframe
+
+    Parameters
+    ----------
+    ak_df : ak.DataFrame
+        Full Arkouda DataFrame containing data to be visualized
+    cols : list
+        (Optional) A specified list of columns to be plotted
+    log : bool
+        use log to scale y
+
+    Notes
+    -----
+    This function displays the plot.
+
+    Examples
+    --------
+    >>> import arkouda as ak
+    >>> from arkouda.plotting import hist_all
+    >>> ak_df = ak.DataFrame({"a": ak.array(np.random.randn(100)),
+                              "b": ak.array(np.random.randn(100)),
+                              "c": ak.array(np.random.randn(100)),
+                              "d": ak.array(np.random.randn(100))
+                              })
+    >>> hist_all(ak_df)
+    """
+
+    if len(cols) == 0:
+        cols = ak_df.columns
+
+    num_rows = int(len(cols) ** 0.5)
+    num_cols = (len(cols) + num_rows - 1) // num_rows
+    fig, axes = plt.subplots(num_rows, num_cols, figsize=(10, 10))
+    fig.tight_layout(pad=2.0)
+
+    if num_rows > 1:
+        axes = axes.flatten()
+    else:
+        axes = [axes]
+
+    for col in cols:
+        try:
+            ax = axes[cols.index(col)]
+            x = ak_df[col]
+
+            if x.dtype == "float64":
+                x = x[isnan(x)]
```

Comment:
maybe im not understanding but shouldn't this be `~isnan`? because otherwise `x` is only the `nan` values
```suggestion
            if x.dtype == "float64":
                x = x[~isnan(x)]
```

Reply:
Definitely supposed to be that, I think the black formatting may have screwed it up. Changing now.